app/loc/lang-sv.json

```diff
@@ -82,35 +82,35 @@
     "STR_OFFICIALLY_CLOSED": "Officiellt avslutat",
     "STR_RECORDS_MOVED": "Protokollsanteckningarna överförda",
     "STR_TECNICALLY_CLOSED": "Tekniskt avslutat",
-    "STR_TOPIC_UNKNOWN": "Status information saknas",
-    "STR_INFO_MENUBTN": "Ändra storlek på sidomenyn",
+    "STR_TOPIC_UNKNOWN": "Statusinformation saknas",
+    "STR_INFO_MENUBTN": "Ändra storlek på sidmenyn",
     "STR_INFO_HOMEBTN": "Gå till startvyn",
     "STR_INFO_VIEW_HORIZ": "Vy sida vid sida",
     "STR_INFO_VIEW_VERT": "Vy under varandra",
     "STR_PRESENTATION": "Förslag",
-    "STR_OPEN_NEW_TAB": "Öppna i ett nytt fönster",
-    "STR_DECISIONS": "Behandling och Beslut",
+    "STR_OPEN_NEW_TAB": "Öppna i ny flik",
+    "STR_DECISIONS": "Behandling och beslut",
     "STR_MOTIONS": "Motioner",
     "STR_MOTION": "Motion",
-    "STR_MOTIONS_OPEN_APP": "Motionapplikation",
+    "STR_MOTIONS_OPEN_APP": "Motionsapplikationen",
     "STR_BUDGET_MOTION": "Budgetmotion",
-    "STR_SUPPORTED": "Understödet",
+    "STR_SUPPORTED": "Understödd",
     "STR_VOTINGS": "Omröstningar",
-    "STR_MTG_STATUS_PUBLISHED_PROPOSALS": "@:STR_PUBLIC_PROPOSALS",
-    "STR_PROP_PUB_COUNT": "*Antal av publicerad förslag",
+    "STR_MTG_STATUS_PUBLISHED_PROPOSALS": "Publicerade förslag",
```

Comment:
Ok to have like this, but notice the two strings have an identical translation.
Original `"STR_MTG_STATUS_PUBLISHED_PROPOSALS": "@:STR_PUBLIC_PROPOSALS"`
is correct as well: it's a reference to reuse the value from string `STR_PUBLIC_PROPOSALS`
Its up to which design you want: duplicate strings or reusing.